src/components/svgs/stations/chongqingrt2021-int.tsx

```diff
@@ -0,0 +1,359 @@
+import React from 'react';
+import { RmgFields, RmgFieldsField } from '@railmapgen/rmg-components';
+import { MonoColour } from '@railmapgen/rmg-palette-resources';
+import { useTranslation } from 'react-i18next';
+import { AttrsProps, CanvasType, CategoriesType, CityCode, Theme } from '../../../constants/constants';
+import {
+    NameOffsetX,
+    NameOffsetY,
+    Station,
+    StationAttributes,
+    StationComponentProps,
+    StationType,
+    defaultStationAttributes,
+} from '../../../constants/stations';
+import { MultilineText } from '../common/multiline-text';
+import { ColorField } from '../../panels/details/color-field';
+
+export const LINE_HEIGHT = {
+    zh: 9,
+    en: 4,
+    top: 4 + 1,
+    middle: 0,
+    bottom: 9 + 1,
+};
+
+const ChongqingRT2021IntStation = (props: StationComponentProps) => {
+    const { id, x, y, attrs, handlePointerDown, handlePointerMove, handlePointerUp } = props;
+    const {
+        names = defaultStationAttributes.names,
+        nameOffsetX = defaultChongqingRT2021IntStationAttributes.nameOffsetX,
+        nameOffsetY = defaultChongqingRT2021IntStationAttributes.nameOffsetY,
+        isRapid = defaultChongqingRT2021IntStationAttributes.isRapid,
+        isWide = defaultChongqingRT2021IntStationAttributes.isWide,
+        rapidColor = defaultChongqingRT2021IntStationAttributes.rapidColor,
+        wideDirection = defaultChongqingRT2021IntStationAttributes.wideDirection,
+    } = attrs[StationType.ChongqingRT2021Int] ?? defaultChongqingRT2021IntStationAttributes;
+
+    const onPointerDown = React.useCallback(
+        (e: React.PointerEvent<SVGElement>) => handlePointerDown(id, e),
+        [id, handlePointerDown]
+    );
+    const onPointerMove = React.useCallback(
+        (e: React.PointerEvent<SVGElement>) => handlePointerMove(id, e),
+        [id, handlePointerMove]
+    );
+    const onPointerUp = React.useCallback(
+        (e: React.PointerEvent<SVGElement>) => handlePointerUp(id, e),
+        [id, handlePointerUp]
+    );
+
+    const getTextOffset = (oX: NameOffsetX, oY: NameOffsetY) => {
+        if (isWide) {
```

Comment:
Better be consistent in using specific attributes. Either `const getTextOffset = () => {` or `const getTextOffset = (oX: NameOffsetX, oY: NameOffsetY, isWide: boolean, wideDirection: 'vertical' | 'horizontal') => {`.